NAudioTests/WaveProviders/MixingWaveProvider32Tests.cs

```diff
@@ -0,0 +1,24 @@
+﻿using NAudio.Wave;
+using NUnit.Framework;
+
+namespace NAudioTests.WaveProviders
+{
+    [TestFixture]
+    public class MixingWaveProvider32Tests
+    {
+        [Test]
+        public void CannotAddSameInputInputProviderSecondTime()

```

Comment:
I believe this is a typo

```suggestion
        public void CannotAddSameInputProviderSecondTime()

```